rconweb/api/instances.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os
+import redis
+from rconweb.settings import ENVIRONMENT
+
+INSTANCE_KEY_PREFIX = "rcon_instance_{}"
+
+def _get_main_redis_db():
```

Comment:
Ahaha so it worked out of the box? I expected I left some mistakes

Reply:
Took me a while to work out what StricRedis was but just importing and fromUrl seems to do the job.